config/initializers/ember.rb

```diff
@@ -1,5 +1,3 @@
 EmberCLI.configure do |c|
-  c.app :exam_builder,
-    bower_path: "bin/bower",
-    exclude_ember_deps: ["jquery", "handlebars"]
+  c.app :exam_builder
```

Comment:
Yeah, sorry, I didn't do a perfect job of minimizing, since I wasn't even getting error messages from Ember-CLI-Rails. Thanks for finding and fixing these discrepancies.